retrieval/client/fsm_test.go

```diff
@@ -42,7 +41,7 @@ type mockClientEnvironment struct {
 }
 
 func (e *mockClientEnvironment) OpenDataTransfer(ctx context.Context, to peer.ID, proposal *deal.Proposal) (datatransfer.ChannelID, error) {
-	return datatransfer.ChannelID{ID: datatransfer.TransferID(rand.Uint64()), Responder: to, Initiator: testnet.GeneratePeers(1)[0]}, e.OpenDataTransferError
+	return datatransfer.ChannelID{ID: datatransfer.TransferID(rand.Uint64()), Responder: to}, e.OpenDataTransferError
 }
 
```

Comment:
@tchardin not sure about this one

Reply:
@gallexis just make a util function to generate a random peer? You just need a dummy peer ID